recipes/libx11/all/conanfile.py

```diff
@@ -0,0 +1,60 @@
+import os
+import subprocess
+import shlex
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+class LibX11Conan(ConanFile):
+    name = "libx11"
+    license = "X11"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.x.org/wiki/"
+    description = "Client interface to the X Window System, otherwise known as \'Xlib\'",
+    settings = "os", "compiler", "build_type", "arch"
+    exports = ["COPYING.txt"]
+    _required_system_package = "libx11-dev"
+    _system_package_tool = None
+
+    def _system_packages(self):
+        if not self._system_package_tool:
+            self._system_package_tool = tools.SystemPackageTool()
+        return self._system_package_tool
+
+    def system_requirements(self):
+        installer = self._system_packages()
+        if not installer.installed(self._required_system_package):
+            raise ConanInvalidConfiguration(
+                "{0} system library missing. Install {0} in your system with something like: "\
+                "sudo apt-get install {0}"
+                .format(self._required_system_package))
+
+    def configure(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration(
+                "This library is only compatible with Linux")
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def _populate_cpp_info(self, lib_name):
+        def get_value(arg, prefix):
+            return arg[len(prefix):] if arg[:len(prefix)] == prefix else ""
+
+        ret = subprocess.check_output("pkg-config --cflags --libs {}".format(lib_name), shell=True)
```

Comment:
Out of curiosity, why not using `tools.PkgConfig` for this ?

Reply:
Yes, that would be using tools.PkgConfig, I'll update it to use it. Thanks!